RPGManager_development/test/unit_test/character_sheet_unit_tests/GetDefaultLanguagesForEachBackgroundUnitTests.java

```diff
@@ -0,0 +1,74 @@
+				{ Background.DALISH_ELF, new LanguagesSet(Languages.ELVEN, Languages.TRADE_TONGUE) },
+				{ Background.DWARF_DUSTER, new LanguagesSet(Languages.DWARVEN_LANGUAGE, Languages.TRADE_TONGUE) },
+				{ Background.ELF_APOSTATE, new LanguagesSet(Languages.TRADE_TONGUE) },
+				{ Background.ELF_CIRCLE_MAGE, new LanguagesSet(Languages.ANCIENT_TEVENE, Languages.TRADE_TONGUE) },
+				{ Background.ESCAPED_ELVEN_SLAVE, new LanguagesSet(Languages.TEVENE, Languages.TRADE_TONGUE) },
+				{ Background.FERELDAN_CRAFTSMEN, new LanguagesSet(Languages.TRADE_TONGUE) },
+				{ Background.FERELDAN_FREEMAN, new LanguagesSet(Languages.TRADE_TONGUE) },
+				{ Background.FERELDAN_NOBLE, new LanguagesSet(Languages.TRADE_TONGUE) },
+				{ Background.FREE_MARCHER, new LanguagesSet(Languages.TRADE_TONGUE) },
+				{ Background.HIGH_BORN_DWARF, new LanguagesSet(Languages.DWARVEN_LANGUAGE, Languages.TRADE_TONGUE) },
+				{ Background.HUMAN_APOSTATE, new LanguagesSet(Languages.TRADE_TONGUE) },
+				{ Background.HUMAN_CIRCLE_MAGE, new LanguagesSet(Languages.ANCIENT_TEVENE, Languages.TRADE_TONGUE) },
+				{ Background.LOW_BORN_DWARF, new LanguagesSet(Languages.DWARVEN_LANGUAGE, Languages.TRADE_TONGUE) },
+				{ Background.NEVARRAN_ADVENTURER, new LanguagesSet(Languages.ORLESIAN, Languages.TRADE_TONGUE) },
+				{ Background.ORLESIAN_COMMONER, new LanguagesSet(Languages.ORLESIAN, Languages.TRADE_TONGUE) },
+				{ Background.ORLESIAN_EXILE, new LanguagesSet(Languages.ORLESIAN, Languages.TRADE_TONGUE) },
+				{ Background.ORLESIAN_NOBLE, new LanguagesSet(Languages.ORLESIAN, Languages.TRADE_TONGUE) },
+				{ Background.ORLESIAN_STUDENT, new LanguagesSet(Languages.ORLESIAN, Languages.TRADE_TONGUE) },
+				{ Background.QUNARI_BERESAAD, new LanguagesSet(Languages.QUNLAT, Languages.TRADE_TONGUE) },
+				{ Background.RIVAINI_MERCHANT, new LanguagesSet(Languages.RIVAINI, Languages.TRADE_TONGUE) },
+				{ Background.SEHERON_CONVERT, new LanguagesSet(Languages.QUNLAT, Languages.TEVENE, Languages.TRADE_TONGUE) },
+				{ Background.SURFACE_DWARF, new LanguagesSet(Languages.DWARVEN_LANGUAGE, Languages.TRADE_TONGUE) },
+				{ Background.TAL_VASHOTH, new LanguagesSet(Languages.QUNLAT, Languages.TRADE_TONGUE) },
+				{ Background.TEVINTER_ALTUS, new LanguagesSet(Languages.TEVENE, Languages.TRADE_TONGUE) },
+				{ Background.TEVINTER_LAETAN, new LanguagesSet(Languages.TEVENE, Languages.TRADE_TONGUE) },
+				{ Background.TEVINTER_SOPORATI, new LanguagesSet(Languages.TEVENE, Languages.TRADE_TONGUE) },
+				{ Background.WAKING_SEA_RAIDER, new LanguagesSet(Languages.ANTIVAN, Languages.TRADE_TONGUE) }, });
+	}
+
+	@Parameter(0)
+	public Background background;
+
+	@Parameter(1)
+	public LanguagesSet expectedLanguage;
+
+	@Test
+	public void testGetDefaultBackgroundForLanguages() {
+		CharacterSheet testCharacterSheet = new CharacterSheet("TestCharacterSheet");		
+		testCharacterSheet.setData(Fields.BASECLASS, background.getAllowedBaseClasses().get(0));

```

Comment:
kódszépészkedés No.2:
Ezt a `background.getAllowedBaseClasses().get(0)`-t ki lehetne emelni egy privát függvénybe amiből kiderül hogy mit is akarsz ezzel csinálni.
pl.:
```java
private getFirstValidBaseClassByBackground(Background background){
    return background.getAllowedBaseClasses().get(0);
}
```